questions.md

```diff
@@ -0,0 +1,2 @@
+1. the concept of REMOTE TRACKING BRANCH should be clarified
```

Comment:
👋 Hi @monicareggiani-unipd ! Thank you for the input. This is something we hope to address in the next iteration of the learning modules.